src/npm.js

```diff
@@ -56,14 +56,28 @@ export async function getDownloads(pkgs) {
     ...pkgsNamesChunks.map(pkgsNames =>
       got(`${c.npmDownloadsEndpoint}/point/last-month/${pkgsNames}`, {
         json: true,
-      }).catch(e => suppressError(e, pkgsNames, { body: {} }))
+      }).catch(error =>
+        suppressError({
```

Comment:
Can we use: suppressError => logWarning and things => type?

Reply:
Good idea, will refactor :)